store/dispatcher_test.go

```diff
@@ -157,5 +157,9 @@ func (n *nullEvent) Node() (format.Node, error) {
 	return nil, nil
 }
 
+func (n *nullEvent) Type() core.ActionType {
```

Comment:
Just returning Delete for now. Perhaps it is worth defining a `core.Noop` ActionType though for future usage, tests, etc?